packages/manager/apps/web/client/app/hosting/multisite/delete/hosting-multisite-delete.controller.js

```diff
@@ -25,14 +25,44 @@ angular.module('App').controller(
         name: 'web::hosting::multisites::detach-domain',
       });
 
+      this.domain = null;
+      this.subDomain = null;
+      this.autoConfigureAvailable = null;
+
+      if (
+        this.$scope.currentActionData.domain.indexOf('.') ===
+        this.$scope.currentActionData.domain.lastIndexOf('.')
+      ) {
+        this.domain = this.$scope.currentActionData.domain;
+      } else {
+        this.subDomain = this.$scope.currentActionData.domain.slice(
+          0,
+          this.$scope.currentActionData.domain.indexOf('.'),
+        );
+        this.domain = this.$scope.currentActionData.domain.slice(
+          this.subDomain.length + 1,
+        );
+      }
```

Comment:
```suggestion
      const domain = this.$scope.currentActionData.domain
      if (domain.indexOf('.') === domain.lastIndexOf('.')) {
        this.domain = domain;
      } else {
        this.subDomain = domain.slice(0, domain.indexOf('.'));
        this.domain = domain.slice(this.subDomain.length + 1);
      }
```

Reply:
It's applied with const { domain } = this.$scope.currentActionData